pkg/retry/retry.go

```diff
@@ -109,5 +109,11 @@ func isErrnoRetryable(e error) bool {
 	case syscall.ECONNREFUSED, syscall.EINTR, syscall.EAGAIN, syscall.EBUSY, syscall.ENETDOWN, syscall.ENETUNREACH, syscall.ENETRESET, syscall.ECONNABORTED, syscall.ECONNRESET, syscall.ETIMEDOUT, syscall.EHOSTDOWN, syscall.EHOSTUNREACH:
 		return true
 	}
+	switch err := e.(type) { //nolint
```

Comment:
Please document what the `//nolint` comment is disabling.

Reply:
Removed.